src/commands/rollMap.js

```diff
@@ -35,7 +35,7 @@ export function rollMap () {
 
   return {
     handler,
-    triggers: ['rollmap', 'rollMap', 'maproll', 'mapRoll'],
```

Comment:
Again, please leave the triggers as they were. I don't think there is any risk of using any of these aliases for any other command in the future.